vue-frontend/src/components/partials/ScreenFooter2.vue

```diff
@@ -36,20 +36,20 @@
             :class="
               socialMedia.image
                 ? ' tw-mt-[20px] sm:tw-my-0 tw-mx-1 tw-w-[50px] tw-h-[50px] tw-cursor-pointer'
-                : ' gradient-btn sm:tw-my-0  tw-mx-1 tw-w-[50px] tw-h-[50px] tw-cursor-pointer tw-rounded-[50%] tw-p-[0.7rem] '
+                : ' tw-w-[50px] tw-h-[50px] sm:tw-mb-0 gradient-btn sm:tw-my-0  tw-mx-1 tw-cursor-pointer tw-rounded-[50%] tw-pt-2 '
             "
           >
             <a :href="socialMedia.url" target="_blank">
               <img
-                class="active:tw-scale-[0.98] tw-h-full tw-w-full"
+                class="active:tw-scale-[0.98] sm:tw-h-full sm:tw-w-full"
                 v-if="socialMedia.image"
                 :src="hover ? designRushHover : designRush"
                 @mouseover="hover = true"
                 @mouseleave="hover = false"
               />
               <font-awesome-icon
                 v-else
-                class="fab tw-h-full tw-w-full"
+                class="fab tw-w-[23px] tw-h-[25px] tw--ml-1 tw-mt-1"
```

Comment:
Remove extra dash from `tw--ml-1`
I think we should not use fixed width and height for this design

Reply:
its negative margin. This height and width is given to resize that icons as it was looking large.